src/Pages/Detail/Comment.scss

```
@@ -1,59 +1,61 @@
 .hiddenInput {
-    display: block;
-    margin-top: 7px;
-    margin-right: 30px;
-    padding: 10px 15px;
+  display: block;
+  margin-top: 7px;
+  margin-right: 30px;
+  padding: 10px 15px;
+  border: 1px solid #e2e2e2;
+  .inputWrapper {
+    position: relative;
+    height: 30px;
     border: 1px solid #e2e2e2;
-    .inputWrapper {
-      position: relative;
-      height: 30px;
-      border: 1px solid #e2e2e2;
-      input {
-        border: none;
-        width: 715px;
-        font-size: 13px;
-        height: 28px;
+    input {
+      border: none;
+      width: 715px;
+      font-size: 13px;
+      height: 28px;
+    }
+    button {
+      position: absolute;
+      border: 1px solid #dadada;
+      width: 50px;
+      height: 25px;
+      margin-top: 2px;
+      background-color: white;
+      color: #7b7b7b;
+    }
+  }
+}
+ul {
+  margin-top: 7px;
+  li {
+    display: flex;
+    margin-bottom: 8px;
+    .commentBox {
+      display: flex;
+      font-size: 13px;
+      .commentWriter {
+        width: 60px;
+        font-weight: bold;
       }
-      button {
-        position: absolute;
+    }
+    .commentButtonBox {
+      opacity: 0;
+      transition: all 0.5s;
```

```suggestion
      transition: opacity 0.5s;
```
투명도가 바뀌는걸 명시해주는게 좋습니다